.github/workflows/cross-images.yml

```diff
@@ -15,7 +15,7 @@ on:
 jobs:
   docker-dev:
     name: Build cross image for ${{ matrix.target }} target
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04 # Stick to 22.04 because of ppa:deadsnakes/ppa is missing smth for 24.04.
```

Comment:
It's building docker images. I cannot understand how the host os affects the docker image. Did I miss something?

Reply:
This is obsolete and was by mistake :)